cmd/wallet/export.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 var exportCmd = &cobra.Command{
-	Use:   "export <name>",
+	Use:   "export <name> [flags]",
```

Comment:
This seems to be redundant. Existing output:

```
$ oasis w export
Error: accepts 1 arg(s), received 0
Usage:
  oasis wallet export <name> [flags]

Flags:
  -h, --help   help for export

Global Flags:
      --config string   config file to use
```

```suggestion
	Use:   "export <name>",
```